save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/organizations/OrganizationSettingsMenu.kt

```diff
@@ -174,22 +172,23 @@ private fun organizationSettingsMenu() = FC<OrganizationSettingsMenuProps> { pro
                             }
                         }
                         classes = "btn btn-sm btn-danger"
-                        modalButtons = { action, window, childrenBuilder ->
+                        modalButtons = { action, closeWindow, childrenBuilder, isClickMode, _ ->
+                            val word = if (isClickMode) "ban" else "delete"
```

Comment:
Please rename `word` to smth clearer, e.g.: `action` or `actionName`

Reply:
Done